buzzmoon-frontend/buzzmoon-vite/src/Components/CreateGame/CreateGame.jsx

```diff
@@ -86,7 +91,18 @@ export default function CreateGame() {
         </button>
 
       </div>
-      <div className="submit-details" />
+      <div className="submit-details">
+        <button type="button" name="upload-game-button" 
+          onClick={async () => {
+            const message = await BackendActor.uploadGame(BackendActor.prepareGameData(title, desc, questions));
+
+            if(message === "Success"){
+              navigate("/compete");
+            }
+          }}>
+          Upload to Server
```

Comment:
Upload to the server might not make a lot of sense to the average user. I suggest you label this as "Create Game" or something more suggestive instead